src/lambdas/api/api.yaml

```diff
@@ -562,7 +562,7 @@ components:
             max: '2019-07-10T13:44:56Z'
           'sci:citation':
             - 'Copernicus Sentinel data [Year]'
-          'eo:gsd':
+          'gsd':
```

Comment:
gsd used to be in the EO extension, now it's part of the STAC Commons fields that have no prefix.